File: src/main/java/com/ruthlessjailer/api/theseus/task/delete/handler/QueuePriority.java
package com.ruthlessjailer.api.theseus.task.delete.handler;

/**
 * @author RuthlessJailer
 */
public enum QueuePriority {

	/**
	 * Gets added to the start of the queue.<p>
	 * Highest priority.
	 */
	IMMEDIATE,
	/**
	 * Gets added to the end of the queue.<p>
	 * Standard priority.
	 */
	NORMAL,
	/**
	 * Gets added to the secondary queue.<p>
	 * Lowest priority.
	 */
	SECONDARY

}

File: src/main/kotlin/com/ruthlessjailer/api/theseus/task/handler/QueuePriority.kt
package com.ruthlessjailer.api.theseus.task.handler

/**
 * @author RuthlessJailer
 */
enum class QueuePriority {
	/**
	 * Gets added to the start of the queue.
	 *
	 *
	 * Highest priority.
	 */
	IMMEDIATE,

	/**
	 * Gets added to the end of the queue.
	 *
	 *
	 * Standard priority.
	 */
	NORMAL,

	/**
	 * Gets added to the secondary queue.
	 *
	 *
	 * Lowest priority.
	 */
	SECONDARY
}